File: Examples/tests/languageIntegration/pythonIntegration/printFO.py
print """
functions {
    pythonDynamicOutput
    {
        type pythonIntegration;

        startCode "print 'Dynamic Starting'";
        executeCode "print 'Dynamic Executing'";
        endCode "print 'Never been here'";
        //        tolerateExceptions true;
        parallelMasterOnly true;
        useNumpy false;
        useIPython false;
    }
    pythonDynamicOutput2
    {
         $pythonDynamicOutput;
         parallelMasterOnly false;
         isParallelized true;
    }
}
"""

File: Examples/tests/languageIntegration/luaIntegration/printFO.lua
print([[
functions {
    luaDynamicOutput
    {
        type luaIntegration;

        startCode "print('Dynamic Starting')";
        executeCode "print('Dynamic Executing')";
        endCode "print('Never been here')";
        //        tolerateExceptions true;
        parallelMasterOnly true;
    }
    luaDynamicOutput2
    {
         $luaDynamicOutput;
         parallelMasterOnly false;
         isParallelized true;
    }
}
]])
